src/software/simulated_tests/kickoff_friendly_play_test.cpp

```diff
@@ -0,0 +1,75 @@
+#include "software/ai/hl/stp/play/kickoff_friendly_play.h"
+
+#include <gtest/gtest.h>
+
+#include "software/simulated_tests/simulated_test_fixture.h"
+#include "software/simulated_tests/validation/validation_function.h"
+#include "software/test_util/test_util.h"
+#include "software/time/duration.h"
+#include "software/time/timestamp.h"
+#include "software/world/world.h"
+
+class KickoffFriendlyPlayTest : public SimulatedTest
+{
+};
+
+TEST_F(KickoffFriendlyPlayTest, test_kickoff_friendly_play)
+{
+    World world = ::Test::TestUtil::createBlankTestingWorld();
+
+    world = ::Test::TestUtil::setEnemyRobotPositions(
+        world,
+        {
+            Point(0.6, 0),
+            Point(0.2, 2.5),
+            Point(0.2, -2.5),
+            world.field().enemyGoalCenter(),
+            world.field().enemyDefenseArea().negXNegYCorner(),
+            world.field().enemyDefenseArea().negXPosYCorner(),
+        },
+        Timestamp::fromSeconds(0));
+
+    world = ::Test::TestUtil::setFriendlyRobotPositions(world,
+                                                        {
+                                                            Point(-3, 2.5),
+                                                            Point(-3, 1.5),
+                                                            Point(-3, 0.5),
+                                                            Point(-3, -0.5),
+                                                            Point(-3, -1.5),
+                                                            Point(-3, -2.5),
+                                                        },
+                                                        Timestamp::fromSeconds(0));
+    world.mutableFriendlyTeam().assignGoalie(0);
+
+    world.mutableBall() = Ball(Point(0, 0), Vector(0, 0), Timestamp::fromSeconds(0));
+
+    std::vector<ValidationFunction> validation_functions = {
+        // This will keep the test running for 9.5 seconds to give everything enough
```

Comment:
any reason why its 9.5 and not 10 or 9? just curious

Reply:
wasn't sure if the test would fail if I made it 10, since it would match the timeout. This will be fixed once there's a proper test implemented here.